profiler/options_test.go

```diff
@@ -62,6 +62,20 @@ func TestOptions(t *testing.T) {
 		assert.Equal(t, "serviceName", cfg.service)
 	})
 
+	t.Run("WithSite", func(t *testing.T) {
+		var cfg config
+		WithSite("some.other.site")(&cfg)
+		assert.Equal(t, "https://intake.profile.some.other.site/v1/input", cfg.apiURL)
+	})
+
+	t.Run("WithSite/override", func(t *testing.T) {
```

Comment:
Fake subtest? 😄

Reply:
Sorry, I don't follow. This subtest makes sure that `WithSite` will override `DD_SITE`

Reply:
I was just teasing. It's all good here. When you write sub-tests they get displayed like this. So this will look as if you have a test called `WithSite` and a subtest called `override`, but that's not true.

Reply:
I see, I see. That's how I did the others as well. :)
Maybe in the future I should stick with `-` in subtest names to avoid confusion.

Reply:
This is all great. No need to change anything.